bundle/language_en.properties

```
@@ -1327,6 +1327,11 @@ angal.patient.newpatient.title
 angal.patient.nextkin                                                                                  = Next of Kin
 angal.patient.no.btn                                                                                   = No
 angal.patient.note                                                                                     = Note
+angal.patient.consensus.consensus.txt                                                                  = Consent to treatment
+angal.patient.consensus.consensus.mandatory.msg                                                        = "Consent to treatment" is mandatory.
+angal.patient.consensus.administrative.txt                                                             = Administrative consent
+angal.patient.consensus.service.txt                                                                    = Consent for additional services
+angal.patient.consensus.border                                                                         = Informed consent
```

```suggestion
angal.patient.consensus.border                                                                         = Informed Consent
```
The border labels have the first letter of each word as a capital letter.